tenets/codelingo/code-review-comments/go-error-fmt/example.go

```diff
@@ -1,8 +1,12 @@
 // Package main is an example package
 package main
 
+import "fmt"
+
 func main() {
 	fmt.Errorf("something bad.")
 	fmt.Errorf("this is okay")
 	fmt.Errorf("This is not okay")
+	fmt.Errorf("THIS is okay")
+	fmt.Errorf("THIS is not okay.")
```

Comment:
```suggestion
	fmt.Errorf("API call thing.")
	fmt.Errorf("THIS is not okay.")
```

wouldn't this currently rewrite to `aPI call thing.`? That's surely bad.

Reply:
That line is not okay because of the trailing full stop. That line will only be changed to remove that. "THIS" will remain as-is

Reply:
Is that what you meant?